bigc/resources/orders.py

```diff
@@ -110,6 +110,14 @@ def create_shipment(self, order_id: int, *, order_address_id: int, items: list[d
 
         return self._api.v2.post(f'/orders/{order_id}/shipments', json=payload)
 
+    def update_shipment(self, order_id: int, shipment_id: int, **kwargs: dict) -> dict:
+        """Updates an order shipment for the specified order"""
+        payload = {
+            **kwargs
+        }
+
+        return self._api.v2.put(f'/orders/{order_id}/shipments/{shipment_id}', json=[payload])[0]
```

Comment:
I could be wrong but I'd think it'd be something like this. 
```suggestion
        return self._api.v2.put(f'/orders/{order_id}/shipments/{shipment_id}', json=kwargs)
```